services/autoscaling/tests/unit/test_dynamic_scaling_task.py

```diff
@@ -0,0 +1,69 @@
+# pylint: disable=no-value-for-parameter
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+import asyncio
+from unittest import mock
+
+import pytest
+from fastapi import FastAPI
+from pytest_mock.plugin import MockerFixture
+from pytest_simcore.helpers.utils_envs import EnvVarsDict
+from simcore_service_autoscaling.core.settings import ApplicationSettings
+
+_FAST_POLL_INTERVAL = 1
```

Comment:
Can you lower further this value, in theory it should still work. I would go for 0.01

Reply:
why would I do that? to what end?

Reply:
Say you need to wait 1 second for this event to trigger. In order to have reliable timeouts, you would need to wait a unit of magnitude (x 10) more than the timeout . With a 0.01 or 0.1 the waiting times would be much smaller. Tests will run faster.